cypress/utils/commands.js

```diff
@@ -5,13 +5,19 @@
 
 // --- Typed commands --
 
-Cypress.Commands.add('getElementByTestId', (testId, options = {}) => {
-  return cy.get(`[data-test-subj="${testId}"]`, options);
-});
+Cypress.Commands.add(
+  'getElementByTestId',
+  (testId, options = { timeout: Cypress.config('defaultCommandTimeout') }) => {
+    return cy.get(`[data-test-subj="${testId}"]`, options);
+  }
+);
 
-Cypress.Commands.add('getElementByTestIdLike', (testId, options = {}) => {
-  return cy.get(`[data-test-subj*="${testId}"]`, options);
-});
+Cypress.Commands.add(
+  'getElementByTestIdLike',
+  (testId, options = { timeout: Cypress.config('defaultCommandTimeout') }) => {
```

Comment:
there should be a way to just type the `options` so that it has all the things that the usual `options` for `cy.get()` has. This way if we ever need to add more stuff in the future, we don't need to explictely update the typing

Reply:
i think you already set this value at the cypress config level so i was kind of shooting in the dark with this one and can prolly remove not positive if the option will be passed down nested levels would make sense to me. but basically anyone could override it. unless you are trying to say it would be better to have a set an options that we would let people set and not let them configure it past that